### DISCUSSION TITLE
Time Table for New Requested Feature Creation

### DISCUSSION THREAD

**Discussion by acgrant** [mui/mui-x#6515] [Archive]
I have a [ticket](https://github.com/mui/mui-x/issues/5368) that I created Jun 30th (reopened August 25th by one of the MUI devs), which was marked as _New Feature_. I was wondering if there's some threshold of votes that has to be reached for it to be put at the top of the queue or whether you have your own triage system in place to assure all the new features get implemented in some finite amount of time.

**Comment by oliviertassinari** [comment]
> if there's some threshold of votes that has to be reached for it to be put at the top of the queue

@acgrant Not threshold, we usually do it relative to the other issues open, we try to implement the features that will have the most impact.

> I was wondering if there's some threshold of votes that has to be reached for it to be put at the top of the queue or whether you have your own triage system in place to assure all the new features get implemented in some finite amount of time.

This happens on GitHub on the issue. If there are no updates, likely it's because it's not on our radar of priorities.

I have locked this discussion as we are not taking any community input on the GitHub discussion, it was meant for polls. I have also closed the Polls category.

**Reply by acgrant** [reply]
Ah got you, thank you for the clarification, excuse the misplacement.
